server/internal/infrastructure/mongo/asset.go

```diff
@@ -49,6 +57,10 @@ func (r *assetRepo) FindByIDs(ctx context.Context, ids id.AssetIDList) ([]*asset
 }
 
 func (r *assetRepo) FindByProject(ctx context.Context, id id.ProjectID, uFilter repo.AssetFilter) ([]*asset.Asset, *usecasex.PageInfo, error) {
+	if !r.f.CanRead(id) {
+		return nil, usecasex.EmptyPageInfo(), nil
+	}
+
 	var filter interface{} = bson.M{
```

Comment:
```suggestion
	filter := bson.M{
```

Reply:
Whenever I do this suggested change I get this error:
```
cannot use mongox.And(filter, "", (bson.M literal)) (value of type interface{}) as primitive.M value in assignment: need type assertion
```

Reply:
oh ok